.github/workflows/close-stale.yml

```diff
@@ -20,3 +20,19 @@ jobs:
           exempt-issue-labels: never-stale
           exempt-pr-labels: never-stale
           any-of-labels: information-requested
+      - uses: actions/stale@v9
+        with:
+          days-before-issue-stale: 0
+          days-before-issue-close: 14
+          stale-issue-message: 'This package does not have an assigned owner and is considered unmaintained. As such this package is in feature-freeze and this issue will be closed with 14 days unless a new owner or a sponsor (@open-telemetry/javascript-approvers) for the feature is found. It is the responsibility of the author to find a sponsor for this component.'
+          close-issue-message: 'This issue was closed because no owner or sponsor has been found after 14 days'
+          only-labels: pkg-status:unmaintained,feature-request
+          exempt-issue-labels: bug,has:sponsor
+      - uses: actions/stale@v9
+        with:
+          days-before-pr-stale: 0
+          days-before-pr-close: 14
+          stale-pr-message: 'This package does not have an assigned owner and is considered unmaintained. As such this package is in feature-freeze and this PR will be closed with 14 days unless a new owner or a sponsor (@open-telemetry/javascript-approvers) for the feature is found. It is the responsibility of the author to find a sponsor for this component.'
```

Comment:
```suggestion
          stale-pr-message: 'This package does not have an assigned owner and is considered unmaintained. As such this package is in feature-freeze and this PR will be closed with 14 days unless a new owner or a sponsor (@open-telemetry/javascript-approvers) for the feature is found. It is the responsibility of the author to find a sponsor for this PR.'
```

Reply:
Applied in https://github.com/open-telemetry/opentelemetry-js-contrib/pull/2272/commits/58d00e74251fa582c9da4ec303e5b2aef1a5baf6